nextflow_schema.json

```diff
@@ -78,7 +200,8 @@
                     "hidden": true,
                     "help_text": "Do not load `igenomes.config` when running the pipeline. You may choose this option if you observe clashes between custom parameters and those supplied in `igenomes.config`."
                 }
-            }
+            },
+            "required": ["fasta"]
```

Comment:
I am probably missing something but what is this doing here?

Reply:
I always ran the pipeline with fasta option, I guess I have required it mechanically. 
This is corrected.